DESCRIPTION

```diff
@@ -88,6 +88,7 @@ Imports:
     readxl,
     rlang,
     stringr,
+    testthat (>= 3.0.0),
```

Comment:
I tried it with testthat as a suggested package but got errors when running check as the test function uses `@inhertiFrom`. Whats the way to do this here? use `package::function` notation rather than using the namespace?

Reply:
yes exactly